Change category subject about "announcements" for restrictions (#224)
mdn/mdn-community: README.md

| **Discussion category** | **Category subject** 	|
|---	|---	|

> | 📣  [Announcements](https://github.com/mdn/mdn-community/discussions/categories/announcements) | This category is reserved for MDN Web Docs staff. There are restrictions in place to prevent others from using this category. |

Reviewer: ```suggestion
| 📣  [Announcements](https://github.com/mdn/mdn-community/discussions/categories/announcements) | This category is reserved for MDN Web Docs staff to make any announcements in relation to the MDN project |
```
